daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -50,34 +24,65 @@ Parameter | Description
 `workflowName` | Identify the workflow type
 `instanceId` | Unique value created for each run of a specific workflow
 
+### Request Contents
 
-### Headers
+In the request you can pass along headers:
 
-As part of the start HTTP request, the caller can optionally include one or more `dapr-workflow-metadata` HTTP request headers. The format of the header value is a list of `{key}={value}` values, similar to the format for HTTP cookie request headers. These key/value pairs are saved in the workflow instance’s metadata and can be made available for search (in cases where the workflow implementation supports this kind of search).
-
-
-## HTTP responses
+```json
+{
+  "input": {
+    // arguments to pass to the workflow
+  }
+}
+```
 
-### Response codes 
+### HTTP Response codes
 
 Code | Description
 ---- | -----------
 `202`  | Accepted
 `400`  | Request was malformed
 `500`  | Request formatted correctly, error in dapr code or underlying component
 
-### Examples of response body for each method
+### Response Contents
 
-#### POST start workflow response body
+The API call will provide a response similar to this:
 
-```bash
+```json
+{
   "WFInfo": {
     "instance_id": "SampleWorkflow"
   }
+}
 ```
 
+## Terminate workflow request
 
-#### POST terminate workflow response body
+Terminate a running workflow instance with the given name and instance id.
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/terminate
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Current default is `dapr` for Dapr Workflows
+`workflowName` | Identify the workflow type
+`instanceId` | Unique value created for each run of a specific workflow
+
+### HTTP Response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or underlying component
+
+### Response Contents
+
+The API call will provide a response similar to this:
 
 ```bash
 HTTP/1.1 202 Accepted
```

Comment:
I'm just noticing that the `Content-Type` and `Content-Length` fields in the response looks wrong. Would you mind removing these for us as part of this PR?